snuba/datasets/entities/sessions.py

```diff
@@ -31,6 +31,9 @@ def __init__(self) -> None:
         writable_storage = get_writable_storage(StorageKey.SESSIONS_RAW)
         materialized_storage = get_storage(StorageKey.SESSIONS_HOURLY)
         read_schema = materialized_storage.get_schema()
+        quantiles = tuple(
+            Literal(None, quant) for quant in [0.5, 0.75, 0.9, 0.95, 0.99]
```

Comment:
do these quantiles here have to be in sorted order, or can I use an arbitrary order?

Also, do I have to use the same order in all the places where I use these, which is quite a few actually.

Reply:
I would keep them in the same order everywhere and in sorted order. Whether this is required I am not sure, but it would be fairly confusing if they were not sorted.